nx/lib/nx/defn/grad.ex

```diff
@@ -766,7 +774,7 @@ defmodule Nx.Defn.Grad do
                [:logical_and, :logical_or, :logical_xor, :logical_not] ++
                [:left_shift, :right_shift, :count_leading_zeros, :population_count] ++
                [:floor, :round, :ceil, :sign] ++
-               [:equal, :greater, :greater_equal, :less, :less_equal, :not_equal]
+               [:equal, :greater, :greater_equal, :less, :less_equal, :not_equal, :argsort]
```

Comment:
@josevalim argsort returns a new tensor of constants (the indices corresponding to the sorted positions), which is why it's grad is zero. This is in accordance with Jax.